### ISSUE TITLE
Get the reindex during the rename. Rename cancelled.

### ISSUE THREAD

**Issue by aivus** [orocrm/oro-phpstorm-plugin#5]
Hi.
I got reindex during rename. Phpstorm cancel renaming because of reindex.
I tried to do it few times, but everytime I was getting reindex and cancellation.

It was fixed after disable this plugin.

It happens when "looking for usage" in process.

![2016-07-26_13-25-06](https://cloud.githubusercontent.com/assets/1021703/17134734/92108130-5334-11e6-8c58-0add4791dfee.png)

![2016-07-26_13-27-00](https://cloud.githubusercontent.com/assets/1021703/17134754/ac60c3d8-5334-11e6-91d9-443d249e0eb9.png)

**Comment by psliwa** [comment]
Thanks. I will fix this issue asap.

**Comment by psliwa** [comment]
@aivus Can you provide what version of PhpStorm and the plugin are you using? Are you sure reindexing is triggered during rename? There was an issue where some indexes were constantly reindexed, but I've fixed that. Can you tell me if you try to rename entity field? Is this field used in some kind of yml / xml configuration files? If yes, can you tell me what filename such files have and maybe screenshot of some part of this files where reference to the field is visible? It would be very helpfull because right now I am not able to reproduce this issue.

**Comment by aivus** [comment]
I'll send you STR asap

**Comment by aivus** [comment]
Environment
```
PhpStorm 2016.2
Build #PS-162.1121.38, built on July 12, 2016
...
JRE: 1.8.0_76-release-b216 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

STR:
1. Clone https://github.com/orocommerce/orocommerce-application
2. `git checkout 2e16b4db97c533303bd567980dc09dc13d345c9f` (to be sure that we have same commit)
3. Open in phpstorm and wait until indexation finished
4. Enable oro plugin
5. Open `Oro\Bundle\ApplicationBundle\Model\AbstractModel`
6. Try to rename field (Refactor -> Rename) `entity` in this class

Expected: rename successfully finished
Actual: Refactoring cancelled during renaming

**Comment by psliwa** [comment]
Thanks a lot, I am able to reproduce this issue - it is a half of the success :) It works in IntelliJ 2016.1 which I use during developing, I will try IntelliJ 2016.2 and the newest php plugin.

**Comment by psliwa** [comment]
The problem should be fixed in 1.0.0-alpha14 version, can you check if it is ok?

**Comment by aivus** [comment]
It seems work fine now. Thanks.
